homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -100,18 +109,18 @@ def update(self):
         self._state = state
         self._device_state_attributes = attr
 
-    def alarm_disarm(self, code=None):
+    def alarm_disarm(self):
```

Comment:
Don't change the method signature from the base class.

Reply:
done